### PR TITLE
To do list structure

### PR DESCRIPTION
**Pull request by J2ZROMERO**
### important! execute npm start in your terminal to see the project with styles and elements added dynamically.

> Create webpack
- [Use of this documentation to create the set-up](https://webpack.js.org/guides/getting-started/#basic-setup)

- [Add webpack-plugin to generate its own index.html](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin)

- [Add style to webpack](https://webpack.js.org/guides/asset-management/#loading-css)

- [Add local server](https://webpack.js.org/guides/development/#using-webpack-dev-server)

> Add index.html as a template.
> Add function to iterate tasks array.
> Add event load to create 3 tasks dynamically.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Jhosep, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have an appropriate ReadMe. Kindly ensure that your ReadMe is detailed and reflects your project. I suggest you use this [recommended template](https://github.com/microverseinc/readme-template/blob/master/README.md). Kindly request for another review when fix is done.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import './style.css';
+import enter from './img/enter.png';
+import refresh from './img/refresh.png';
+import dots from './img/dots.png';
+
+document.querySelector('.refresh').src = refresh;
+document.querySelector('.enter').src = enter;
+const contentTask = document.querySelector('.pctasks');
+
+const tasktR = [{ completed: true, description: 'go to run', index: 0 }, { completed: false, description: 'go to eat', index: 2 }, { completed: true, description: 'go to sleep', index: 3 }];
```
Comment (prolajumokeoni) [inline src/index.js:10]: - [ ] Kindly change the value for completed to false, based on the requirements the value for completed will be false 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @J2ZROMERO,
 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @J2ZROMERO  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Changes implemented ✔️ 



 Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**